cron.py

```diff
@@ -18,36 +20,45 @@ def sync_get(url):
     except requests.RequestException:
         return False
 
+
 def ping_every_30_minutes():
-    engine = create_engine('sqlite:///tools.db')
+    engine = create_engine("sqlite:///tools.db")
     SessionInit = sessionmaker(bind=engine)
     session = SessionInit()
-    tools = session.query(Tool).all()
+    if os.path.exists(".venv"):
+        print(
+            "Running in local envoirnment, limiting health status check upto 10 tools"
+        )
+        tools = session.query(Tool).limit(10).all()
+    else:
+        print("Running in production envoirnment, checkig health status for all tools")
+        tools = session.query(Tool).all()
```

Comment:
we use virtual env in production server also, so the `.venv` file would exist on the prod server leading to running of only the first condition all the time :(